sru/release-32/test-user-config-created.sh

```diff
@@ -0,0 +1,78 @@
+#!/bin/bash
+set -e
+
+series=$1
+install_from=$2 # either path to a .deb, or 'staging', or 'proposed'
+
+name=$series-dev
+
+function cleanup {
+  lxc delete $name --force
+}
+
+function on_err {   
+  echo -e "Test Failed"
+  cleanup
+  exit 1
+}
+trap on_err ERR
+
+
+lxc launch ubuntu-daily:$series $name
+sleep 5
+
+# Install latest ubuntu-advantage-tools
+lxc exec $name -- apt-get update > /dev/null
+lxc exec $name -- apt-get install  -y ubuntu-advantage-tools > /dev/null
+echo -e "\n* Latest u-a-t is installed"
+echo "###########################################"
+lxc exec $name -- apt-cache policy ubuntu-advantage-tools
+echo -e "###########################################\n"
+
+# Create user-config.json file with the given content
+lxc exec $name -- sh -c "echo '{\"http_proxy\": \"http://someuser:somepassword@example.com:3128\"}' > /var/lib/ubuntu-advantage/user-config.json"
+if lxc exec $name -- test -e /var/lib/ubuntu-advantage/user-config.json; then
+  user_config_contents=$(lxc exec $name -- cat /var/lib/ubuntu-advantage/user-config.json)
+  http_proxy_value=$(echo "$user_config_contents" | jq -r '.http_proxy')
+fi
```

Comment:
This is cool, but we already know the expected value - we don't need to parse it from the file.
```suggestion
http_proxy_value="http://someuser:somepassword@example.com:3128
# Create user-config.json file with the given content
lxc exec $name -- sh -c "echo '{\"http_proxy\": \"$http_proxy_value\"}' > /var/lib/ubuntu-advantage/user-config.json"
```